solidity/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @hyperlane-xyz/core
 
+## 3.6.0
+
+### Patch Changes
+
+- @hyperlane-xyz/utils@3.6.0
+
```

Comment:
Why are these changelog changes in the PR? Typically changesets does this for us in one of the "Version packages" PRs it makes automatically

Reply:
i might have "yarn version:prepare" by mistake, reverting...